README.rst

```diff
@@ -37,6 +37,82 @@ on `PyPI`_ and `GitHub`_.
 .. _`PyPI`: https://pypi.org/project/pymedphys/
 .. _`GitHub`: https://github.com/pymedphys/pymedphys
 
+Statement of need
```

Comment:
Do we really need this in our readme file?

Reply:
I don't think we need the PyPi link as the current version is autogenerated in the header
I'm unsure about github
Either way it's not hurting anything and the descriptions make sense so either way works for me

Reply:
I'm a strong believer in pointing to the source code (Github for PyMedPhys).
I benefitted from gcc and gdb back in the day, and had to hack it to work on HP-UX 6.5...
So... If someone uses pip to install PyMedPhys, will the readme be available and point them in the right direction to find the source?
If not, where can the user determine where to find the source?  I realise that the source is in fact embedded in the package, but finding fixes that are not yet released is one reason for hunting down the repo...

Reply:
Sorry, I was pointing to the long "statement of need" (see the first line of green).

@Matthew-Jennings, do we really need this whole statement of need here?